utils/airtable.js

```diff
@@ -0,0 +1,26 @@
+import Airtable from 'airtable';
+
+const { AIRTABLE_API_KEY: API_KEY } = process.env;
+const { AIRTABLE_USER_ID: USER_ID } = process.env;
```

Comment:
If these keys don't exist in the environment, the app crashes. Could you fix this?

Reply:
Can you also add them to:
- the readme file
- `.env.default` (without the values)
- `run.sh`